packages/fast-usdc/src/exos/advancer.js

```diff
@@ -271,6 +277,7 @@ export const prepareAdvancerKit = (
         borrowerFacet: M.remotable(),
         poolAccount: M.remotable(),
         intermediateRecipient: M.opt(ChainAddressShape),
+        settlementAddress: M.opt(ChainAddressShape),
```

Comment:
minor: should remove M.opt

Reply:
Thanks for looking it over again. e49ad004bd coming in another PR